py-polars/polars/functions/lazy.py

```diff
@@ -1734,7 +1734,8 @@ def collect_all(
     """
     Collect multiple LazyFrames at the same time.
 
-    This runs all the computation graphs in parallel on Polars threadpool.
+    This runs all the computation graphs in parallel on the Polars threadpool,
+    and returns the collected DataFrames in the same order as input.
```

Comment:
This is an ideal candidate for the `Returns` section, right?

Reply:
You make a compelling case ;)
Moved as suggested.